dev/io.openliberty.jakarta.xmlBinding.4.0/resources/io/openliberty/jakarta/xmlbinding/resources/JAXBMessages.nlsprops

```diff
@@ -0,0 +1,30 @@
+###############################################################################
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+
+# NLS_MESSAGEFORMAT_ALL
+#
+#CMVCPATHNAME io.openliberty.jakarta.xmlbinding.4.0/resources/io/openliberty/jakarta/xmlbinding/resources/JAXBMessages.nlsprops
+#ISMESSAGEFILE TRUE
+#COMPONENTPREFIX CWWKW
+#COMPONENTNAMEFOR CWWKW WebSphere XML Binding Tools Code
+#RANGE 1400 - 1499
+#Message range shared with com.ibm.ws.jaxb.tools, com.ibm.ws.jaxb.tools.2.3, io.openliberty.xmlbinding.3.0.internal.tools, and io.openliberty.jakarta.xmlBinding.4.0
+#
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+
+warning.property.map.jaxbcontextfactory.not.found=CWWKW1404W: The property, jakarta.xml.bind.JAXBContextFactory, was found in Properties map and set to {0}, but the specificied implementation cannot be loaded. The default reference implementation will be used.
+warning.property.map.jaxbcontextfactory.not.found.explanation=The JAXBContextFactory is set to an alternate implementation but the specified XML Binding implementation is not available on the application class path.
```

Comment:
Change "The JAXBContextFactory" to "The JAXBContextFactory property" or to "JAXBContextFactory"